src/main/java/org/jabref/gui/exporter/ExportAction.java

```diff
@@ -33,6 +33,9 @@
 
     private static final Log LOGGER = LogFactory.getLog(ExportAction.class);
 
+    private ExportAction() {
```

Comment:
Ah okay, good to know, seems like they are different exportActions here. 
Some time ago I was working with these action classes and therefore was just crurious.